geometry_msgs/Point point
float32 time